### PR TITLE
Amaan/create password vault item view

### PR DESCRIPTION
**Pull request by 9naama**
Password Vault Item view, main view UI work, and some changes to copy credentials.

### CODE REVIEW

File: mystyle.xml
Comment (hydrabeer) [inline mystyle.xml:1]: Why is this part of the PR?

**Review by hydrabeer — CHANGES_REQUESTED** [review]
Adherence to CA looks great. Can merge with more tests included

File: src/main/java/service/home/interface_adapter/HomePresenter.java [hunk 1]
```diff
@@ -1,23 +1,58 @@
...
+                this.viewManagerModel.setState(ViewConstants.PASSWORD_VAULT_ITEM_VIEW);
+                this.viewManagerModel.onStateChanged();
+            }
+            //            case "noteItem" -> {
+            //                noteVaultItemViewModel.setState(
+            //                        new NoteVaultItemState((NoteVaultItem) item)
+            //                );
+            //                noteVaultItemViewModel.onStateChanged();
+            //            }
+            default -> throw new RuntimeException("Unhandled vault item type: " + item.getType());
```
Comment (HACKERALERT) [inline src/main/java/service/home/interface_adapter/HomePresenter.java:47]: I think in the exception folder there is an InvalidVaultItemException that may be more descriptive here

File: src/main/java/views/HomeView.java [hunk 2]
```diff
@@ -62,68 +67,189 @@ public void actionPerformed(ActionEvent e) {
...
+        vaultItemPanel.add(subTitle, BorderLayout.SOUTH);
+        vaultItemPanel.add(accessButton, BorderLayout.EAST);
+        return vaultItemPanel;
+    }
 
-        userInfo.setForeground(Color.WHITE);
-        userInfo.setFont(new DoorkeyFont());
-        userInfo.setAlignmentX(CENTER_ALIGNMENT);
+    private JButton addAccessButton(int height, AbstractVaultItem vaultItem) {
+        final JButton accessButton = new DoorkeyButton.DoorkeyButtonBuilder("\uD83D\uDD13")
```
Comment (HACKERALERT) [inline src/main/java/views/HomeView.java:154]: add a cmoment to descrie what \uD83D\uDD13 is

**Review by HACKERALERT — APPROVED** [review]

**Review by saeedshrouf — APPROVED** [review]
Good job on making requested changes with testing, architecture looks really good also.

**Review by hydrabeer — APPROVED** [review]

**Pull request merged**